.lando.yml

```diff
@@ -91,6 +91,6 @@ events:
   post-db-import:
     - appserver: drush cc all
   post-rebuild:
-    - appserver: drush en devel -y
-    - appserver: drush dl field_sql_norevisions --dev
-    - appserver: drush en field_sql_norevisions -y
+    - appserver: drush --root=/app/web en devel -y
```

Comment:
This fixes this error:

```
Command pm-enable needs a higher bootstrap level to run - you will need to invoke drush from a more functional Drupal environment to run this command.
                                  [error]
The drush command 'en devel' could not be executed.                                                                                                                                                                                                  
                                  [error]
ERROR ==> The drush command 'en devel' could not be executed.                                                                                                                                                                                                  
                                            [error] 
```

Reply:
Ugh, that's correct but should be provided earlier by the tooling stanza for drush. Come on, Lando!